### PR TITLE
Cleanup templates & pin deck.gl 8.4.10

### PR DESCRIPTION
**Pull request by VictorVelarde**
Small cleanup on both templates

- [x] remove duplicated addLayer in stores
- [x] pin deck.gl 8.4.10
- [x] clean skeleton app:
    - [x] remove isoLine code
    - [x] remove geocoderWidget & layer
    - [x] remove Login button from header

### CODE REVIEW

File: template-skeleton/template/src/components/views/login/UserMenu.js [hunk 1]
```diff
@@ -34,12 +33,12 @@ function UserMenu() {
     return null;
   }
 
-  // User is NOT logged in, so display Login with CARTO
+  // User is NOT logged in, so display nothing
   if (!user) {
-    return <UserMenuLogin />;
+    return null;
```
Comment (AdriSolid) [inline template-skeleton/template/src/components/views/login/UserMenu.js:38]: `return;` is fine?

**Review by AdriSolid — CHANGES_REQUESTED** [review]

File: template-skeleton/template/src/components/views/login/UserMenu.js (hunk 1 above)
Comment (VictorVelarde) [reply template-skeleton/template/src/components/views/login/UserMenu.js:38]: I don't have a preference. Is there any official recommendation on that?

Comment (AdriSolid) [reply template-skeleton/template/src/components/views/login/UserMenu.js:38]: I saw that we return `null` in several components, for me we can leave it like this :+1:

**Review by AdriSolid — APPROVED** [review]
:D

**Review by AdriSolid — APPROVED** [review]
:+1:

**Pull request merged**